source/js/components/share-button-group/share-button-group.jsx

```diff
@@ -20,7 +20,7 @@ export default class ShareButtonGroup extends React.Component {
   renderFacebookButton() {
     let label =
       this.props.version === `mini` ? (
-        <span class="sr-only">Share on Facebook</span>
+        <span className="sr-only">Share on Facebook</span>
```

Comment:
React uses `className`

Reply:
omg we still had some of those left? =_=